lib/logging/monitor.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Logging
+  class Monitor < ::ZeroSilentFailures::Monitor
+    def initialize(service)
+      @service = service
+      super(@service)
+    end
+
+    ##
+    # log GET request
+    #
+    # @param message [String]
+    # @param metric [String]
+    # @param form_type [String]
+    # @param user_account_uuid [User]
+    #
+    def track_request(message, metric, form_type = 'Unknown Form Type', user_account_uuid = nil, call_location: nil)
```

Comment:
instead of `form_type` have it be 'context' so what ever gets sent will just be appended to the logger payload.
also probably will need to allow 'tags' to be given, but those should be required